client-v2/integration/server/server.js

```diff
@@ -110,25 +112,45 @@ module.exports = async () =>
       return res.json(result);
     };
 
-    // Attempts at a capture group:
-    // /api/(preview|live)/*
-    // /api/(?:preview|live)/
     app.get('/api/live/*', handler);
     app.get('/api/preview/*', handler);
 
     app.get('/config', (_, res) => res.json(config));
-    app.get('/collection/:id', (_, res) => res.json(collection));
```

Comment:
This GET route was not being used - we actually use a 'POST` request to handle requests for collections.

Reply:
👍